openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1507,7 +1507,96 @@ It is up to the Credential Issuer whether to update both the signature and the c
 
 # Privacy Considerations
 
-TBD
+When [@!RFC9396] is used, the Privacy Considerations of that specification also apply.
+
+The privacy principles of [@ISO.29100] should be adhered to.
+
+## User Consent
+
+The Credential Issuer should obtain the End-User's consent before issuing Credential(s)
+to the Wallet. It should be made clear to the End-User what information is being included in the
+Credential(s) and for what purpose.
+
+## Minimum Disclosure
+
+To ensure minimum disclosure and prevent Verifiers from obtaining claims unnecessary for the
+transaction at hand, when issuing Credentials that are intended to be created once and then used a
+number of times by the End-User, the Credential Issuers and the Wallets SHOULD implement
+Credential formats that support selective disclosure, or consider issuing a separate Credential
+for each user claim.
+
+## Storage of the Credentials
+
+To prevent a leak of End-User data, especially when it is signed, which risks revealing private data of End-Users to
+third parties, systems implementing this specification SHOULD be designed to minimize the amount
+of End-User data that is stored. All involved parties SHOULD store Verifiable Credentials
+containing privacy-sensitive data only for as long as needed, including in log files. Any logging of
+End-User data should be carefully considered as to whether it is necessary at all. The time logs are retained
+for should be minimized.
+
+After Issuance, Credential Issuers SHOULD NOT store the Issuer-signed Credentials if they
+contain privacy-sensitive data. Wallets SHOULD store Credentials only in encrypted form, and,
+wherever possible, use hardware-backed encryption. Credentials that are no longer valid to present should be deleted as soon as
+possible.
```

Comment:
```suggestion
After Issuance, Credential Issuers SHOULD NOT store the Issuer-signed Credentials if they
contain privacy-sensitive data. Wallets SHOULD store Credentials only in encrypted form, and,
wherever possible, use hardware-backed encryption. Wallet should delete Credentials that are
no longer valid to present as soon as possible.
```

Reply:
One minor nit, is there any reason to caveat these normative requirements as only applying to credentials with "privacy-sensitve data" or could we generalise these statements a little more by removing that part

e.g

```suggestion
After Issuance, Credential Issuers SHOULD NOT store the Issuer-signed Credentials. Wallets SHOULD store Credentials only in encrypted form, and,
wherever possible, use hardware-backed encryption. Credentials that are no longer valid to present should be deleted as soon as
possible.
```